src/main/java/com/hackaton/model/ResultsModel.java

```diff
@@ -0,0 +1,105 @@
+package com.hackaton.model;
+
+import com.hackaton.interfaces.ResultInterface;
+
+import javax.swing.table.AbstractTableModel;
+import java.util.*;
+
+public class ResultsModel extends AbstractTableModel {
+
+    private List<ResultInterface> data;
+    private final Object[] COLUMNS = {"Nazwa", "Ilość"};
+
+    public ResultsModel() {
```

Comment:
Konstruktor do wywalenia później.

Reply:
Spoko, niech na razie wypełnia to wszystko przykładowymi danymi